amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-studio.adoc

```diff
@@ -38,6 +39,8 @@ Installing the connector adds it to a Studio project. Anypoint Studio provides t
 
 Configure an input source for the connector, such as a connector operation, HTTP Listener, or Schedule.
 
+If you use the Receive Messages input source in cluster mode and want to receive messages on the primary node only, select the `Primary node only` field of the `Advanced` sub-tab on the *Receive messages tab*. For more information, see xref:amazon-sqs-connector-config-topics.adoc#node-behavior[Receiving Messages in Cluster Mode].
```

Comment:
```suggestion
If you use the Receive Messages input source in cluster mode and want to receive messages on the primary node only, select the `Primary node only` field of the `Advanced` sub-tab on the *Receive messages* tab. For more information, see xref:amazon-sqs-connector-config-topics.adoc#node-behavior[Receiving Messages in Cluster Mode].
```

Reply:
I would also add the steps as I mentioned in the Additional Configuration topic.
_. In Studio, select the Receive Messages input source of your flow.
.  On the *Advanced* sub-tab of the *Receive messages* tab, select the *Primary node only* checkbox._